libraries/botbuilder-dialogs-adaptive/src/generators/templateEngineLanguageGenerator.ts

```diff
@@ -50,7 +50,12 @@ export class TemplateEngineLanguageGenerator<T = unknown, D extends Record<strin
 
     public generate(dialogContext: DialogContext, template: string, data: D): Promise<T> {
         try {
-            const result = this.lg.evaluateText(template, data);
+            // BUGBUG: I'm casting objects to <any> to work around a bug in the activity factory.
```

Comment:
I don't fully understand this issue, can you link me to the casting?

Reply:
this issue is added by someone else. it is removed in the main branch. Sorry for the ambiguity.